client/client.go

```diff
@@ -1865,3 +1867,24 @@ func (c *client) WatchGlobalConfig(ctx context.Context) (chan []GlobalConfigItem
 	}()
 	return globalConfigWatcherCh, err
 }
+
+func (c *client) ReportMinResolvedTS(ctx context.Context, storeID, minResolvedTS uint64) error {
+	if span := opentracing.SpanFromContext(ctx); span != nil {
```

Comment:
maybe we can panic directly? I think only TiKV can use it.

Reply:
how about removing it?